h/h_api/exceptions.py

```diff
@@ -0,0 +1,61 @@
+from h.h_api.model.json_api import JSONAPIError, JSONAPIErrorBody
+
+
+class JSONAPIException(Exception):
+    """An exception which can turn itself into a a JSON API error response."""
+
+    http_status = None
+
+    def _error_bodies(self):
+        """Get the instances of JSONAPIErrorBody representing this error."""
+        raise NotImplementedError()
+
+    def as_dict(self):
+        """Get a JSON API compatible dict representing this error."""
+        return JSONAPIError.create(self._error_bodies()).raw
+
+
+class SchemaValidationError(JSONAPIException):
+    """Represent a number of schema validation errors.
```

Comment:
```suggestion
    """A schema validation error.
```

Reply:
It is more than one, so I've renamed the class and updated the string.

Reply:
Nah chickened out. It's a single case of you not matching the schema in potentially many different ways. I've changed it to:

```The provided data did not match the schema.```